src/Components/Publishing/Display/NewDisplayCanvas.tsx

```diff
@@ -0,0 +1,44 @@
+import { Flex } from "@artsy/palette"
+import { ErrorBoundary } from "Components/ErrorBoundary"
+import { isHTLAdEnabled } from "Components/Publishing/Ads/EnabledAd"
+import { AdDimension, AdUnit } from "Components/Publishing/Typings"
+import React, { SFC } from "react"
+import styled from "styled-components"
+
+interface DisplayCanvasProps {
+  adUnit?: AdUnit
+  adDimension?: AdDimension
+}
+
+export const NewDisplayCanvas: SFC<DisplayCanvasProps> = props => {
+  const { adUnit, adDimension } = props
+
+  if (!isHTLAdEnabled()) {
+    return null
+  }
+
+  return (
+    <ErrorBoundary>
+      <DisplayCanvasContainer>
```

Comment:
This should follow https://github.com/artsy/reaction/pull/2413/files#diff-9a095ec4c410c988e7febdfec25eefd5R22 in terms of inlining props